tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -107,6 +103,16 @@ def delete_app(self, app_manifest):
         from gaiatest.apps.homescreen.regions.confirm_dialog import ConfirmDialog
         return ConfirmDialog(self.marionette)
 
+    def swipe_screen(self):
+        el = self.marionette.find_element(*self._homescreen_icon_page_locator)
+        action = Actions(self.marionette)
+        action.flick(el, 100, 100, 100, 100, 200)
+        action.perform()
+        #Actions(self.marionette).\
+        #    flick(el, 0, 0, 500, 500, 200).\
+        #    perform()
+
```

Comment:
wahouu thank you So much @mwargers that's I was looking for but I didn't find the right way to do it :)